vassal-app/src/main/java/VASSAL/script/ExpressionInterpreter.java

```diff
@@ -658,6 +658,9 @@ else if (src instanceof GameModule) {
     else if (src instanceof Map) {
       mapList.add((Map) src);
     }
+    else if (src instanceof ReportState.OldAndNewPieceProperties) {
```

Comment:
Could that cause an NPE if the piece isn't on a map?

Reply:
I've added some general NPE protection further down

Reply:
`null instanceof Whatever` is always false, so it should not.